gatsby-config.ts

```diff
@@ -274,7 +272,7 @@ module.exports = {
     {
       resolve: `gatsby-source-filesystem`,
       options: {
-        path: `${__dirname}/src/assets/locales`,
+        path: `./src/assets/locales`,
```

Comment:
I infer that is because you encountered path-resolution issues after updating to latest gatsby-version?
In that case you can also use `process.cwd()` instead